### PR TITLE
fix failed v2v function test case

### PR DESCRIPTION
**Pull request by musuemu**
Signed-off-by: liuzi <zili@redhat.com>

### CODE REVIEW

**Comment by musuemu** [comment]
#  avocado run --vt-type v2v --vt-machine-type q35 function_test_esx.positive_test.with_proxy.esx_60.rhev.rhv_upload.it_vddk.esx_uri --vt-connect-uri qemu:///system
JOB ID     : dcaa0adfaf2b6146d42f82cbf300617da4b2f5e4
JOB LOG    : /root/avocado/job-results/job-2019-11-07T15.10-dcaa0ad/job.log
 (1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.with_proxy.esx_60.rhev.rhv_upload.it_vddk.esx_uri: PASS (1762.20 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 1763.14 s

**Comment by musuemu** [comment]
failed case link:
https://libvirt-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/v2v-RHEL-7.8-runtest-x86_64-function-function_test_esx-rhev/2/testReport/junit/rhev/function_test_esx/positive_test_fstab_comma_esx_65_rhev_rhv_upload_it_vddk_vpx_uri/

#  avocado run --vt-type v2v --vt-machine-type q35 function_test_esx.positive_test.fstab_comma.esx_65.rhev.rhv.it_vddk.esx_uri --vt-connect-uri qemu:///system
JOB ID     : 36b148e29cc362d6f15f09bf011b13a104c0ea12
JOB LOG    : /root/avocado/job-results/job-2019-11-07T16.06-36b148e/job.log
 (1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.fstab_comma.esx_65.rhev.rhv.it_vddk.esx_uri: PASS (846.69 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 847.61 s

File: v2v/tests/cfg/function_test_esx.cfg [hunk 1]
```diff
@@ -205,7 +205,7 @@
             only esx_65
             checkpoint = fstab_comma
             main_vm = VM_NAME_ESX_FSTAB_SEPBY_COMMA
-            msg_content = 'virt-v2v: warning: mount: mount exited with status 32: mount: .*? already mounted .*?\(ignored\)'
```
Comment (xiaodwan) [inline v2v/tests/cfg/function_test_esx.cfg]: https://github.com/avocado-framework/avocado-vt/pull/2321

After my above pr is merged, you can simply change 'mount: .*?' to 'mount:.*?' and run the case again.

**Comment by musuemu** [comment]
# avocado run --vt-type v2v --vt-machine-type q35 function_test_esx.positive_test.fstab_comma.esx_65.rhev.rhv.it_vddk.esx_uri --vt-connect-uri qemu:///system
JOB ID     : a81a959c9a0070a7fea37f9b03a221471bf17852
JOB LOG    : /root/avocado/job-results/job-2019-11-12T17.18-a81a959/job.log
 (1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.fstab_comma.esx_65.rhev.rhv.it_vddk.esx_uri: PASS (1152.03 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 1152.88 s

File: v2v/tests/cfg/function_test_esx.cfg [hunk 2]
```diff
@@ -205,7 +205,7 @@
...
+            msg_content = 'virt-v2v: warning: mount: mount exited with status 32: mount:.*? is already mounted on .*?\(ignored\)'
```
Comment (xiaodwan) [inline v2v/tests/cfg/function_test_esx.cfg]: >>> re.search(ptn, buf7).group(0)
'virt-v2v: warning: mount: mount exited with status 32: mount: /dev/sda2 is \nalready mounted or /sysroot/home busy\n       /dev/sda2 is already mounted on /sysroot/home (ignored)'
>>> re.search(ptn, buf8).group(0)
'virt-v2v: warning: mount: mount exited with status 32: mount: \n/sysroot/home: /dev/sda2 already mounted on /sysroot/home. (ignored)'

You change doesn't work in rhel8. Please modify it as I said, only remove the space in 'mount: .*?'

**Pull request merged**